apps/docs/features/docs/Reference.sections.tsx

```diff
@@ -297,13 +297,43 @@ async function ApiEndpointSection({ link, section, servicePath }: ApiEndpointSec
 
   return (
     <RefSubLayout.Section columns="double" link={link} {...section}>
-      <StickyHeader title={endpointDetails.summary} className="col-[1_/_-1]" />
+      <StickyHeader
+        title={
+          <>
+            {endpointDetails.summary}
+            {endpointDetails.deprecated && (
+              <span
+                className="
+                  uppercase
+                  whitespace-nowrap align-middle inline-block -translate-y-0.5
+                  border border-amber-700 bg-amber-300 text-amber-900
+                  rounded-full font-mono font-medium text-xs px-2 py-0.5 ml-2"
+              >
+                deprecated
+              </span>
+            )}
+          </>
+        }
+        className="col-[1_/_-1] gap-0.75em"
```

Comment:
```suggestion
        className="col-[1_/_-1]"
```

gap here won't do anything since it's not a flex/grid/multicolumn, it's the margin-left providing the spacing

Reply:
Left it by accident, thanks